helm-chart/renku-core/values.yaml

```diff
@@ -5,9 +5,14 @@
 replicaCount: 1
 
 cacheDirectory: /svc/cache
-
+cleanupInterval: 60
```

Comment:
Is this in seconds? If so, it seems a bit low to me. Maybe we should use a longer default.

Reply:
Yes, it's seconds. I would say that that the rate of cleanup checks is sufficient to do it once a minute, but you're right that default object TTL should probably be bigger. I've expanded it to 30 minutes (as a default). All of those values can be changed and tinkered with during runtime to adjust it to the usage pattern.